src/internal.c

```diff
@@ -17564,7 +17564,7 @@ int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
                         #ifndef NO_RSA
                             case rsa_sa_algo:
                             {
-                                if (verifySig == NULL) {
+                                if (verifySig == NULL && ssl->sigLen > 0) {
```

Comment:
Better to add check to consider ssl->sigLen == 0 an error. Same for 17613 below.

Recommend something like:
```
if (ssl->sigLen == 0) {
    ERROR_OUT(BAD_COND_E, exit_sske);
}
```